app/models/comment.rb

```diff
@@ -26,7 +26,7 @@ class Comment < ApplicationRecord
   has_ancestry cache_depth: true
   has_paper_trail
 
-  belongs_to :commentable, polymorphic: true, optional: true
+  belongs_to :commentable, polymorphic: true, optional: true, counter_cache: true
```

Comment:
Rails' counter cache method documentation has changed and now requires us to define it on the `belongs_to` relationship.